### ISSUE TITLE
Next Earth Lab deep learning meetup: Friday, Feb 28, 2020 11-noon

### ISSUE THREAD

**Issue by mikoontz** [earthlab/meetup#86]
Dear Deep Learnéd colleagues,

Our next weekly meetup will be next Friday, February 28, 2020 from 11 to noon in SEEC C325 (same room that we met in on Feb. 21st). I've reserved this room at this time slot through the May 22, so we can plan to meet there each week.

For next week, please work through [Lesson 2](https://course.fast.ai/videos/?lesson=2). It will likely take a minimum of 2 hours. Please check back to this issue to get the tips and tricks for properly linking your Google Drive if you are using the Colab approach! Max Cook and Adam will lead us through the lesson. If there is time, we can preview [Lesson 3](https://course.fast.ai/videos/?lesson=3).

If you have any specific questions about getting Lesson 2 working, feel free to add a comment to this issue and we can crowd source a solution.

Remember the resources folder is on Google Drive [here](https://drive.google.com/drive/folders/1PLrND4qQWcdM3L1gM_UJ6drqJ6kCHyB4?usp=sharing). Check out the paper that Max Joseph recommended in the 'papers' subdirectory.

See you Friday, February 28th in SEEC C325 or at [https://cuboulder.zoom.us/j/2375826693](https://cuboulder.zoom.us/j/2375826693) at 11am!

**Comment by joemcglinchy** [comment]
already an argument for this on the interwebs: Are Nick Nolte and Gary Busey the same person?

![image](https://user-images.githubusercontent.com/4762214/75574408-757d8a80-5a1b-11ea-8666-510a0c998f40.png)

**Comment by joemcglinchy** [comment]
validation error_rate vs training error_rate: https://forums.fast.ai/t/error-rate-for-training-set/41746

tl;dr
you can always ask for `data.validate(ds_type=DatasetType.train)` when the training is finished

**Comment by mikoontz** [comment]
Thanks for a great meeting today, everyone! And thanks to Adam and Max for leading.

Notes from today are here: [https://docs.google.com/document/d/1y8cjkFJl7pDe4cCjqcrqLV6gmd0ddF7L8cri0pH5y7A/edit#](https://docs.google.com/document/d/1y8cjkFJl7pDe4cCjqcrqLV6gmd0ddF7L8cri0pH5y7A/edit#)

Keep an eye out for an issue describing next week's tasks.

**Comment by admahood** [comment]
Now that's a good model idea, nick nolte vs gary busey
